asteroid/models/base_models.py

```diff
@@ -99,7 +109,7 @@ def file_separate(
             sf.write(save_name, est_src, fs)
 
     def _separate(self, wav, *args, **kwargs):
-        """ Hidden separation method
```

Comment:
Did not mean to commit this... but I wonder why it hasn't already been formatted with black?